padertorch/ops/sequence/mask.py

```diff
@@ -0,0 +1,66 @@
+import torch
+
+
+def compute_mask(x, sequence_lengths, batch_axis=0, sequence_axis=1):
+    """
+    This function calculates a mask which indicates the position of non-padded values.
+    It can be used to do subsequent operations only on non-padded values.
+    >>> x, seq_len = 2*torch.ones((3,1,10,4)), [1, 2, 3]
+    >>> mask = compute_mask(x, sequence_lengths=seq_len, batch_axis=0, sequence_axis=-1)
+    >>> mask[:,0]
+    tensor([[[1., 0., 0., 0.],
+             [1., 0., 0., 0.],
+             [1., 0., 0., 0.],
+             [1., 0., 0., 0.],
+             [1., 0., 0., 0.],
+             [1., 0., 0., 0.],
+             [1., 0., 0., 0.],
+             [1., 0., 0., 0.],
+             [1., 0., 0., 0.],
+             [1., 0., 0., 0.]],
+    <BLANKLINE>
+            [[1., 1., 0., 0.],
+             [1., 1., 0., 0.],
+             [1., 1., 0., 0.],
+             [1., 1., 0., 0.],
+             [1., 1., 0., 0.],
+             [1., 1., 0., 0.],
+             [1., 1., 0., 0.],
+             [1., 1., 0., 0.],
+             [1., 1., 0., 0.],
+             [1., 1., 0., 0.]],
+    <BLANKLINE>
+            [[1., 1., 1., 0.],
+             [1., 1., 1., 0.],
+             [1., 1., 1., 0.],
+             [1., 1., 1., 0.],
+             [1., 1., 1., 0.],
+             [1., 1., 1., 0.],
+             [1., 1., 1., 0.],
+             [1., 1., 1., 0.],
+             [1., 1., 1., 0.],
+             [1., 1., 1., 0.]]])
+
+    Args:
+        x:
+        sequence_lengths:
+        batch_axis:
+        sequence_axis:
```

Comment:
Can you add an explanation for these or remove the section? I think that the `Args` and `Returns` sections don't make sense if they don't contain a description of what is going on.

Reply:
E.g., it would be interesting to know that it returns a one-mask if `sequence_legnths is None`.